src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+/* eslint-disable @typescript-eslint/lines-between-class-members */
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+  color: 'red' | 'green' | 'blue';
+  private a: number;
+  private b: number;
+  private c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
```

Comment:
change it for all the cases
```suggestion
  color: Color;
  private a: number;
  private b: number;
  private c: number;

  constructor(
    color: Color,
```